### PR TITLE
Extract sst_importer

### PR DESCRIPTION
**Pull request by brson**
###  What have you changed?

Extract most of the import module to the sst_importer crate, leaving only the ImportSSTService behind, which is still coupled to parts of the server.

This is part of extracting raftstore https://github.com/tikv/tikv/issues/5437, and a follow up to extracting keys https://github.com/tikv/tikv/pull/5376.

###  What is the type of the changes?

- Engineering (engineering change which doesn't change any feature or fix any issue)

###  How is the PR tested?

cargo test, make clippy, make format

###  Does this PR affect documentation (docs) or should it be mentioned in the release notes?

no

###  Does this PR affect `tidb-ansible`?

no

###  Refer to a related PR or issue link (optional)

https://github.com/tikv/tikv/issues/5437

### CODE REVIEW

**Comment by siddontang** [comment]
PTAL @kennytm @overvenus

File: components/sst_importer/src/lib.rs [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+extern crate lazy_static;
+#[macro_use]
+extern crate quick_error;
+#[macro_use]
+extern crate serde_derive;
+#[macro_use(slog_error, slog_warn, slog_info)]
+extern crate slog;
+#[macro_use]
+extern crate slog_global;
+
```
Comment (overvenus) [inline components/sst_importer/src/lib.rs]: Do we use to `extern crate tikv_alloc` explicitly in order to enable custom mallocator?

Comment (brson) [reply components/sst_importer/src/lib.rs]: We do have `extern crate tikv_alloc` in many libraries. I don't know if it's necessary or not, but I'll add it here.

Comment (brson) [reply components/sst_importer/src/lib.rs]: I have done so.

**Comment by brson** [comment]
PTAL @kennytm @overvenus

**Pull request merged**